Providers/AppsFlyerProvider.m

```diff
@@ -30,6 +30,9 @@ - (instancetype)initWithITunesAppID:(NSString *)iTunesAppID key:(NSString *)key
 
 - (void)identifyUserWithID:(NSString *)userID andEmailAddress:(NSString *)email {
     [AppsFlyerTracker sharedTracker].customerUserID = userID;
+    if (email) {
+        [[AppsFlyerTracker sharedTracker] setUserEmails:@[email] withCryptType:EmailCryptTypeSHA1];
```

Comment:
In case you want to encrypt it, and not show it in plain. I changed it to plain.